app/utils/is-a-potential-pan.test.js

```diff
@@ -0,0 +1,28 @@
+'use strict'
+
+const isAPotentialPAN = require('./is-a-potential-pan')
+const { expect } = require('chai')
+
+describe('Is A Potential card number', () => {
+  it('should return true if reference passes luhn check', () => {
+    expect(isAPotentialPAN('4242424242424242')).to.equal(true)
```

Comment:
Should we add another test case for a 15 digit card number that passes the luhn check? Seeing as it's only 15/16 digits we care about it's probably worthwhile.

Reply:
Good spot